package.json

```diff
@@ -110,7 +110,6 @@
     "edition": "octane"
   },
   "volta": {
-    "node": "10.16.3",
-    "yarn": "1.22.4"
+    "node": "12.19.1"
```

Comment:
Should at npm version be pinned here?

Reply:
We can do that yes! Which version should it be?

Reply:
I usually start with latest, `volta pin npm`

Reply:
Pinned an mentioned in the README.